test/consistent-destructuring.js

```diff
@@ -0,0 +1,344 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import {outdent} from 'outdent';
+import rule from '../rules/consistent-destructuring';
+
+const ruleTester = avaRuleTester(test, {
+	parserOptions: {
+		ecmaVersion: 2020
+	}
+});
+
+const invalidTestCase = (code, correctCode) => {
+	return {
+		code,
```

Comment:
```suggestion
		code,
		output: code,
```

Make sure output didn't change.